seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/config/BaseSourceConfigOptions.java

```diff
@@ -134,6 +134,13 @@ public class BaseSourceConfigOptions {
                     .noDefaultValue()
                     .withDescription("To be read sheet name,only valid for excel files");
 
+    public static final Option<String> EXCEL_ENGINE =
+            Options.key("excel_engine")
+                    .stringType()
+                    .noDefaultValue()
```

Comment:
It is recommended to directly give a default value of a POI here for convenience, and make a list of supported types in .md.

Reply:
+1, we can use enum type.

Reply:
OK, I'll edit it.